service/ziti-tunnel/service/state.go

```diff
@@ -223,7 +223,8 @@ func (t *RuntimeState) CreateTun(ipv4 string, ipv4mask int, applyDns bool) (net.
 	}
 	log.Info("routing applied")
 
-	if applyDns {
+	zitiPoliciesEffective := windns.IsNrptPoliciesEffective()
+	if applyDns || !zitiPoliciesEffective {
 		//for windows 10+, could 'domains' be able to replace NRPT? dunno - didn't test it
 		luid.SetDNS(windows.AF_INET, []net.IP{ip}, nil)
```

Comment:
can you output a clear log message at info level indicating that dns is being applied to the TUN and why?